db.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     {
       id: 'NYT',
       name: 'New York Times',
+      isSelected: true,
```

Comment:
Should we add a check that there is exactly 1 `isSelected`?

Reply:
Comment in `db.js` added.